pkg/odo/cli/component/list.go

```diff
@@ -70,9 +70,9 @@ func (lo *ListOptions) Run() (err error) {
 			machineoutput.OutputSuccess(components)
 		} else {
 			w := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
-			fmt.Fprintln(w, "APP", "\t", "NAME", "\t", "PROJECT", "\t", "TYPE", "\t", "SOURCE", "\t", "STATE", "\t", "CONTEXT")
```

Comment:
why do we need to remove the source from display?

Reply:
Because otherwise the source would be empty for local and binary components for commands such as `odo list --path <some_path>`.